src/Tools/AnalyzerRunner/AnalyzerRunner.csproj

```diff
@@ -1,6 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
 <Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\..\..\build\Targets\GenerateCompilerExecutableBindingRedirects.targets" />
```

Comment:
just curious. how this binding redirect works?

Reply:
@jasonmalinowski ⬆️

Reply:
There are core MSBuild targets that will automatically generate binding redirects; the usual process is ResolveAssemblyReferences looks across all your dependencies, figures out the versions, and updates your app.config during build. You can also pass some "hints" in which say "assume I have this version or that version around I need to redirect from". That file just contains those hints.